core/services/versionchooser/utils/chooser.py

```diff
@@ -219,7 +219,7 @@ async def get_available_versions(self, repository: str) -> web.Response:
         for image in await self.client.images.list():
             if not image["RepoTags"]:
                 continue
-            if not any("companion-core" in tag for tag in image["RepoTags"]):
+            if not any(tag.startswith("companion-core:") for tag in image["RepoTags"]):
```

Comment:
Can we also make it valid for blueos-core ?

Reply:
I think that belongs on it's own pr adding support for blueos-core